internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -112,3 +196,28 @@ func getTlsConfiguration(req *s2av2pb.GetTlsConfigurationReq) (*s2av2pb.SessionR
 		return nil, err
 	}
 }
+
+func buildValidatePeerCertificateChainSessionResp(StatusCode uint32, StatusDetails string, ValidationResult s2av2pb.ValidatePeerCertificateChainResp_ValidationResult, ValidationDetails string) *s2av2pb.SessionResp{
+	return &s2av2pb.SessionResp {
+		Status: &s2av2pb.Status {
+			Code: StatusCode,
+			Details: StatusDetails,
+		},
+		RespOneof: &s2av2pb.SessionResp_ValidatePeerCertificateChainResp {
+			&s2av2pb.ValidatePeerCertificateChainResp {
+				ValidationResult: ValidationResult,
+				ValidationDetails: ValidationDetails,
+				Context: &s2av2ctx.S2AContext{
+					/* TODO(rmehta19): Fill out context fields.
```

Comment:
@matthewstevenson88 , we can talk offline about this, I was wondering which of the context fields we can expect S2A to fill out. From there, I can fill them out with defaults here, and handle them in cert_verifier.go

Reply:
I think we should set all of them. Please let me know if you need test data that we can use to populate these fields.

Reply:
Removing the context fields for now as noted in another comment made about adding context fields when we construct new certs with cnf values